summit.html

```diff
@@ -21,7 +21,7 @@
         <nav class="header-nav">
           <a class="header-footer-nav-link" href="https://github.com/eiffel-community/eiffel">Protocol</a>
           <a class="header-footer-nav-link" href="/community.html">Community</a>
-          <a class="header-footer-nav-link" href="/summit.html">Eiffel Summit</a>
+          <a class="header-footer-nav-link" href="/summit.html">Eiffel Online Summit</a>
```

Comment:
Should we change this to online? Perhaps it's better to leave the navigation as is and just change the header on this page?

Reply:
missed that, well pointed